nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-connection-configuration.js

```diff
@@ -1526,7 +1608,13 @@
 
                     // handle each prioritizer
                     $.each(connection.prioritizers, function (i, type) {
-                        $('#prioritizer-available').children('li[id="' + type + '"]').detach().appendTo('#prioritizer-selected');
+                        var selectedPrioritizer = $('#prioritizer-available').children('li[id="' + type + '"]');
+
+                        var draggableElement = selectedPrioritizer.find('div.draggable-control');
+                        if (draggableElement.length === 0 ) {
```

Comment:
Is there a scenario where prioritizer from the available list will already have a `draggable-control`?

Reply:
@mcgilman Thanks for reviewing! This check was accidentally leftover from a previous implementation I had and now a prioritizer from the available list will not have a `draggable-control` anymore. I have removed the check. Can you review again? Thanks!